cnf-certification-test/operator/suite.go

```diff
@@ -42,24 +42,22 @@ var _ = ginkgo.Describe(common.OperatorTestKey, func() {
 	var env provider.TestEnvironment
 	ginkgo.BeforeEach(func() {
 		env = provider.GetTestEnvironment()
+		testhelper.SkipIfEmptyAny(ginkgo.Skip, env.Operators)
```

Comment:
I would revert the change here and add this under each it statement. Otherwise, just modifying one TC will require to check all others (when adding new TCs or modifying). It is way simpler to make the skip relevent to a single TC always.

Reply:
Yeah I was 50/50 about moving this originally.  Sounds good I'll switch it back.

Reply:
Fixed!